ISSUES.txt

```diff
@@ -1,365 +0,0 @@
-
```

Comment:
Hmm TBH I'm not sure, that file is really old (last relevant change was in May 10 2014) so I figured I could delete it. In retrospect I should probably have done that in a separate PR, sorry about that. Should I restore it?